src/ngraph/op/experimental/quantized_conv_bias.cpp

```diff
@@ -53,6 +55,10 @@ op::QuantizedConvolutionBias::QuantizedConvolutionBias(const shared_ptr<Node>& d
     auto scale_val = scale_const_op->get_vector<float>();
     this->m_scale = scale_val[0];
 
+    auto bias_scale_const_op = std::static_pointer_cast<ngraph::op::Constant>(bias_scale);
```

Comment:
this is not a safe cast.  at a minimum:  use dynamic cast and check for nullptr.  better yet, keep this as a `Node` and wait to get the `float` scale value in the CPU backend before calling MKLDNN.  Same goes for `scale_val` above.  Same goes for other experimental ops.  Can we get this cleaned up?